lnrpc/rpc.proto

```diff
@@ -839,6 +849,22 @@ message LightningAddress {
     string host = 2 [json_name = "host"];
 }
 
+message EstimateFeeRequest {
+    /// The map from addresses to amounts for the transaction.
+    map<string, int64> AddrToAmount = 1;
+
+    /// The target number of blocks that this transaction should be confirmed by.
+    int32 target_conf = 2;
+}
+
+message EstimateFeeResponse {
+    /// The total fee in satoshis.
+    int64 fee_sat = 1;
```

Comment:
Should include the `json_name` parameters?

Reply:
Hm, I'm not really sure what it changes to be honest, but added! https://github.com/lightningnetwork/lnd/pull/1228/commits/b06f1f01bae9a9b74a45161c525c24f40b7f9f2a

Reply:
Yeah I recently realized that they're not required. The name of the variable will be used as a default.